### ISSUE TITLE
custom_card_afvalophaling not showing as it should be

### ISSUE THREAD

**Issue by cgilis** [UI-Lovelace-Minimalist/UI#475]
**Describe the bug**
If I add this card, I get this (see screenshot)
Did I do something wrong?

**To Reproduce**

**Expected behavior**

**Screenshots**
<img width="511" alt="CleanShot 2022-02-26 at 17 55 58@2x" src="https://user-images.githubusercontent.com/62599539/155851911-600ebba9-7cd7-4d18-b043-6158b511a9ae.png">

**Additional context**
```
- type: "custom:button-card"
  template:
    - card_afvalophaling
  variables:
    ulm_card_ophaling_vandaag: "sensor.recycleapp_recycle_vandaag"
    ulm_card_ophaling_morgen: "sensor.recycleapp_recycle_morgen"
    ulm_card_datum_gft: "sensor.recycleapp_recycle_gft"
    ulm_card_datum_pmd: "sensor.recycleapp_recycle_pmd"
    ulm_card_datum_rest: "sensor.recycleapp_recycle_restafval"
    ulm_card_datum_papier: "sensor.recycleapp_recycle_papier"
```

**Comment by AndyVRD** [comment]
Hi,

Have you seen/read the README.md? I mentioned that "This card can only be used if the integration "Home-Assistant-Sensor-Afvalbeheer" from pippyn is installed". :)

You can use this with other trash collection integrations but the output from these sensors can be different and will have another view in the card. You still can template your sensors to get the same output, example below.

```
sensor:
  - platform: template
    sensors:
      time_formatted_short:
        friendly_name: "sensor_date_short_recycle_today"
        value_template: "{{ as_timestamp(states('sensor.recycleapp_recycle_vandaag')) | timestamp_custom('%d-%m') }}"
```

**Comment by Tommatheussen** [comment]
If you are not dependent on these sensors in other views, automations or components, you can also just switch to the Afvalbeheer integration (https://github.com/pippyn/Home-Assistant-Sensor-Afvalbeheer) as mentioned before. They support the RecycleApp out of the box without the need for additional template sensors ;)

**Comment by cgilis** [comment]
Hi I'm using that integration :-)

![CleanShot 2022-02-28 at 12 14 40](https://user-images.githubusercontent.com/62599539/155974033-486743a9-28e0-46b0-a9fd-4580dbffb09d.png)
![CleanShot 2022-02-28 at 12 14 30](https://user-images.githubusercontent.com/62599539/155974041-75fe483a-269f-464c-b100-0d7bb2aed534.png)

**Comment by Tommatheussen** [comment]
Are you by any chance using any of the optional settings like `dateobject` or `dateformat`? The default configuration for me just works, it looks like your sensors have a datetime state instead of just a date string. Mine for example says `16-03-2022` as sensor state.
Regarding the Glass sensor, I'm looking at adjusting the card myself, as I don't need to GFT part for example, hopefully I'll have time for that this week

**Comment by AndyVRD** [comment]
Hi,

dateformat needs to be...

```
dateformat: '%d-%m'
```

**Comment by cgilis** [comment]
I've changed it to that timeformat, but still this:
(restarted HA after the change)

![CleanShot 2022-02-28 at 13 18 36](https://user-images.githubusercontent.com/62599539/155982404-51c08c1a-5dd1-49c1-bc7b-2674ccf47a43.png)

**Comment by Tommatheussen** [comment]
Pretty sure you'll need to remove (or set to `0`) the `dateobject` property. That one takes precedence over the dateformat

**Comment by cgilis** [comment]
Nice this works:

![CleanShot 2022-02-28 at 13 44 41](https://user-images.githubusercontent.com/62599539/155985642-93b9cf14-c9a1-43cb-84c9-54c9e826ffae.png)

I assume this is only visible when there is an ophaling?
![CleanShot 2022-02-28 at 13 45 18](https://user-images.githubusercontent.com/62599539/155985759-fd963ce1-220d-4f26-abed-a5ed834fb18c.png)

**Comment by cgilis** [comment]
> Hi,
> 
> dateformat needs to be another format (see below). If you can send me the collection state from the glass sensor i can add it. :)
> 
> ```
> dateformat: '%d-%m'
> ```

Hi, how can I share with you the collection state?
Is thesame format as all the other ones..

Wastecollector: recycleapp
Hidden: false
Sort_date: 20220310
Days_until: 10
unit_of_measurement: ''

**Comment by AndyVRD** [comment]
Will add it this weekend + PR.

**Comment by cgilis** [comment]
Nice!!! Let me know how to upgrade, Thx @AndyVRD !

**Comment by cgilis** [comment]
> 

Strange thing, it's not showing up, and there is a collection today:

Screenshot of today:
![CleanShot 2022-03-03 at 08 43 15](https://user-images.githubusercontent.com/62599539/156519297-7011026b-79b1-4857-b917-474c1fb9a3d5.png)

Screenshot of yesterday:
![image](https://user-images.githubusercontent.com/62599539/156519431-238c309f-16bf-4335-9bc4-38326205b850.png)

**Comment by cgilis** [comment]
> @cgilis Yes it only shows wen there is a collection. For the glass collection i meant if this is the only collection that day or is there sometimes another collection on the same day?

Strange thing, it's not showing up, and there is a collection today:

Screenshot of today:
![CleanShot 2022-03-03 at 08 43 15](https://user-images.githubusercontent.com/62599539/156519297-7011026b-79b1-4857-b917-474c1fb9a3d5.png)

Screenshot of yesterday:
![CleanShot 2022-03-03 at 08 47 07](https://user-images.githubusercontent.com/62599539/156519781-8299981b-8405-4bfd-be23-24ab474cb8dd.png)

**Comment by AndyVRD** [comment]
Can you send the state from the today collection sensor? Anyway this weekend I’m going to change some things because for now I’m checking the state of the sensor, for example pmd, restafval so i can template the output to PMD + Restafval. I will change this to just check if the state of tomorrow or today collections is not equal to “Geen” and then display the state but that way it will be pmd, restafval instead of PMD + Restafval.

**Comment by cgilis** [comment]
Here you go:

<img width="613" alt="CleanShot 2022-03-03 at 10 09 29@2x" src="https://user-images.githubusercontent.com/62599539/156532961-dae384cc-ccad-413f-97ff-c9d2b9f61c59.png">

**Comment by AndyVRD** [comment]
I don't have that state for my collections so that's why it not show the collection. Anyway this weekend i will fix it + add the glass collection.

**Comment by cgilis** [comment]
> I don't have that state for my collections so that's why it not show the collection. Anyway this weekend i will fix it + add the glass collection.

Super!!! Thx

**Comment by AndyVRD** [comment]
See #507

**Comment by cgilis** [comment]
Do I just wait when there is an update of the minimalist  theme in hacs? Or how do I continue?

**Comment by AndyVRD** [comment]
You can just wait on a HACS update or copy the code to the yaml file of the card.

**Comment by cgilis** [comment]
Ok super! Will try that! Thank You!

**Comment by AndyVRD** [comment]
Your welcome! :)

PS: @CM000n, can you close this? Thx!

**Comment by cgilis** [comment]
Let me buy you a coffee first ;-) do you have a paypal link?

**Comment by AndyVRD** [comment]
Great! Yes i have PayPal and it's realy kind of you buying me a coffee but just enjoy it and give that few bucks to one of your kids. :)

**Comment by cgilis** [comment]
Thx @AndyVRD , will do!

Btw, I just changed the state in the developer tools to pmd, and gives this result:
<img width="260" alt="CleanShot 2022-03-04 at 13 43 58@2x" src="https://user-images.githubusercontent.com/62599539/156765936-2280119e-1ed3-4c94-ae0c-62de8c9d3c17.png">

Is this another issue? Or did I do something wrong?
Sorry to bother you again

**Comment by AndyVRD** [comment]
From which sensor you changed the state? I just tested it by changing the state of the today or tomorrow sensor to "pmd" and it works fine here.

**Comment by cgilis** [comment]
I've tested both (today and tomorrow), both are giving the same result.

**Comment by AndyVRD** [comment]
If you have Discord dm me so we can troubleshoot the issue.

**Comment by cgilis** [comment]
> If you have Discord dm me so we can troubleshoot the issue.

Super! Will DM you on Discord tomorrow!

**Comment by cgilis** [comment]
Hi Andy, Sorry can we have a chat tomorrow?

Kr

**Comment by AndyVRD** [comment]
Sure, just send me a dm on discord.

**Comment by cgilis** [comment]
@AndyVRD sorry but what is your discord username?

**Comment by AndyVRD** [comment]
> @AndyVRD sorry but what is your discord username?

AndyVR#1021
